deploy/charts/cert-manager/values.yaml

```
@@ -23,7 +23,7 @@ global:
   logLevel: 2
 
   leaderElection:
-    # Override the namespace used to store the ConfigMap for leader election
+    # Override the namespace used for the leader election lease (not Config Map)
```

Stating not `ConfigMap` is superfluous.

```suggestion
    # Override the namespace used for the leader election lease
```

---

Yer I agree, I didn't have it at first and kept changing my mind. Changed and merged commits.